File: 8-kyu/collatz-conjecture-3n-plus-1/collatz-conjecture-3n-plus-1.py
def hotpo(n):
    result = n
    count = 0
    if n == 1:
        return 0
    while result > 1:
        if result % 2 == 0:
            result = result /2
        else :
            result = 3 * result + 1
        count += 1
    return count

File: 8-kyu/collatz-conjecture-3n-plus-1/collatz-conjecture-3n-plus-1.java
public class CollatzConjecture {  
  public static int hotpo(int n) {
    // TODO: Implement the function
    var cnt = 0;
    while (n != 1) {
      if (n % 2 == 0) {
        n = n/2;
      } else {
        n = 3 * n+1;
      }
      cnt ++;
    }
    return cnt;
  } 
}